content/integrations/kubernetes.md

```diff
@@ -316,7 +312,20 @@ As the 5.17.0 release, Datadog Agent now supports built in [leader election opti
 * Unhealthy
 
 ### Service Checks
-The Kubernetes check does not include any service check at this time.
+
+The Kubernetes check includes the following service checks:
+
+* `kubernetes.kubelet.check`:  
+
+If `CRITICAL`, either `kubernetes.kubelet.check.ping` or `kubernetes.kubelet.check.syncloop` is in `CRITICAL` state.
```

Comment:
Sorry, I was a bit terse in my report in #1904.

It is true that should any one or more of these subservice checks enter `CRITICAL` state, the overarching service check will enter `CRITICAL` state; however, additionally -

`kubernetes.kubelet.check` will also go `CRITICAL` if the kubelet is completely unresponsive.  (e.g.: the kubelet is not running.)  In cases like this, _no status_ will be sent for `.ping` or `.syncloop` (or any other instrumented kubelet subsystem added by Kubernetes in the future), but `kubernetes.kubelet.check` should faithfully report the problem by entering `CRITICAL` state.

I think this detail is worth capturing because users may be surprised to find that `.ping` will _not_ transition to `CRITICAL` when the kubelet API is completely unavailable.

Reply:
👌  description updated